vpdq/python/tests/test_util.py

```diff
@@ -33,5 +33,5 @@ def output_hash_to_file(output_hash_filename, hashes):
 
     with open(output_hash_filename, "w") as file:
         for cur in hashes:
-            file.write(f"{cur.frame_number},{cur.quality},{hash_to_hex(cur.hash['w'])}")
+            file.write(f"{cur.frame_number},{cur.quality},{hash_to_hex(cur.hash['w'])},{cur.time_stamp}")
```

Comment:
These floats might be super long. How many digits are useful? Surely no more than 3 (milliseconds)?

Reply:
Forget to do this in python, I thought it was done in CPP. Just found out it was not until the IO.